mods/species/ascent/items/id_control.dm

```diff
@@ -51,9 +53,6 @@
 	if(ispath(id_card))
 		id_card = new id_card(src)
 	. = ..()
-	robotize()
```

Comment:
Why are you removing these lines?

Reply:
Nvm, I'm reading the changes backwards and missed the other lines.
Not completely sure that setting status like that will work but can see what you're shooting for.

Reply:
Its been working fine so far. 
And I saw a bunch of species do set the prothetics flag already in their definition already anyways.